### DISCUSSION TITLE
How make accordion the default layout for new container

### DISCUSSION THREAD

**Discussion by diocletiann** [nikitabobko/AeroSpace#994] [questions-and-answers]
I have `default-root-container-layout = 'accordion'` which works initially, but it seems after a window is "moved out", new windows are tiled, and when moving one window toward another and they are both separate containers, it uses tiled layout.

Stacking then requires `join-with` instead of move and it would be nice for each container to remember `accordion` and just use `move` like in i3 ($mod+direction) when the workspace is set to "stacked" or "tabbed" layout. I also have this in my config per one of the suggestions:
```
after-startup-command = [
    'workspace 2', 'layout v_accordion',
    'workspace 3', 'layout v_accordion',
    'workspace 4', 'layout v_accordion',
    'workspace 5', 'layout v_accordion',
    'workspace 6', 'layout v_accordion',
    'workspace 7', 'layout v_accordion',
    'workspace 8', 'layout v_accordion',
    'workspace 9', 'layout v_accordion',
    'workspace 1', 'layout v_accordion'
]
```

In case I'm not doing this correctly or explaining it well, basically I want a window that is moved in the direction of another to always stack into the destination window's container.

**Comment by LukasKnuth** [comment]
As far as I understand the config, `root-container` relates to the actual root container at the workspace. What you're creating are "child containers".

That said, I'd love a `default-child-container-layout` option for the exact same reason you listed here.
